css/style.css

```diff
@@ -1,34 +1,876 @@
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background-color: rgb(231, 228, 228);
+  color: #6070ff;
+  padding: 0.3rem 0.3rem 0.3rem 0.3rem;
+  border-radius: 8px;
+}
+
+.css {
+  display: flex;
+  margin-right: 15px;
+  position: static;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background-color: rgb(231, 228, 228);
+  color: #6070ff;
+  padding: 0.3rem 0.3rem 0.3rem 0.3rem;
+  border-radius: 6px;
+}
+
+.script {
+  display: flex;
+  margin-right: 15px;
+  position: static;
+  height: 16px;
+  left: 12px;
+  top: 4px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  background-color: rgb(231, 228, 228);
+  color: #6070ff;
+  padding: 0.3rem 0.3rem 0.3rem 0.3rem;
+  border-radius: 6px;
+}
+
+.but {
+  margin: 15px;
+  display: flex;
+  align-self: flex-start;
+  position: static;
+  height: 50px;
+  left: 12px;
+  top: 12px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: white;
+  border: 1px solid blue;
+  border-radius: 16px;
+  padding: 0.7rem;
+}
+
+.lan,
+.frame,
+.skill {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 2rem 1rem 0;
+  margin: 0 1rem 1rem 1rem;
+}
+
+/*
+.framework {
+  border-bottom: solid;
+  border-bottom-color: #000;
+}
+
+.skills {
+  border-bottom: solid;
+  border-bottom-color: #000;
+
+} */
+
+.second-list-tyle {
+  margin-bottom: 0;
+}
+
+.first-list-style {
+  justify-content: space-between;
+}
+
+.but1 {
+  flex-basis: 100%;
+  padding-left: 10px;
+  padding-right: 10px;
+  margin-left: 0;
+  margin-bottom: 30px;
+}
+
+.pro {
+  display: flex;
+  align-items: center;
+  margin: 0 2.5rem 2rem 1rem;
+  background-color: #f7f7f9;
+  border-radius: 1rem;
+  padding: 0.5rem 0 0.5rem 0;
+  text-align: center;
+}
+
+.lang {
+  height: 24px;
+  left: 12px;
+  right: 78px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.jsi {
+  margin: 5px;
+}
+
+.js {
+  margin: 5px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 510;
+  font-size: 15px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  color: #253858;
+}
+
+.footer {
+  background-color: #6070ff;
+  font-style: normal;
+  font-family: Poppins, sans-serif;
+  height: 779px;
+  border-radius: 100px 0 0 0;
+  padding-top: 103px;
+  padding-right: 30px;
+  padding-left: 30px;
+}
+
+.contactme {
+  display: flex;
+  justify-content: center;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure your page is responsive on small-screen devices for accessibility purposes
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199676723-2a34a1e5-e807-4ae2-9199-61ced19b314d.png)

![image](https://user-images.githubusercontent.com/97100091/199676290-46e6e790-2cbd-4a2f-8893-bfab712a0ddc.png)
![image](https://user-images.githubusercontent.com/97100091/199676649-0703abd9-4b97-4f08-9270-18b579167dcb.png)


Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/199676412-ab25f349-47e9-4a56-8e23-6db5a687a72e.png)